File: features/main/src/main/java/com/inquallity/daggersandbox/main/MainEntry.java
package com.inquallity.daggersandbox.main;

import android.content.Context;
import android.content.Intent;

import com.inquallity.daggersandbox.core.Entry;

public class MainEntry implements Entry {


    @Override
    public String getId() {
        return "main";
    }

    @Override
    public void open(Context ctx) {
        ctx.startActivity(new Intent(ctx, MainActivity.class));
    }
}

File: features/main/src/main/java/com/inquallity/daggersandbox/main/MainEntry.kt
package com.inquallity.daggersandbox.main

import android.content.Context
import android.content.Intent
import com.inquallity.daggersandbox.core.Entry
import com.inquallity.daggersandbox.core.crossmodule.MAIN_ENTRY

class MainEntry : Entry {

    override val id: String
        get() = MAIN_ENTRY

    override fun open(ctx: Context) {
        ctx.startActivity(Intent(ctx, MainActivity::class.java))
    }
}